source/qb64pe.bas

```diff
@@ -23690,15 +23734,9 @@ FUNCTION SCase2$ (t$)
             CASE "_ANDALSO"
                 SCase2$ = "_AndAlso"
 
-            CASE "ANDALSO"
-                SCase2$ = "AndAlso"
-
```

Comment:
Ah, you found some more $NOPREFIX remains, thought I catched them all in the meantime. :)